src/Components/NavBar.js

```diff
@@ -1,4 +1,7 @@
 import React from 'react';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import Container from 'react-bootstrap/Container';
```

Comment:
it would look cleaner if you do: `import { Navbar, Nav, Container } from 'react-bootstrap';`